src/javascript/JContent/ContentRoute/ToolBar/BrowseControlBar/BrowseControlBar.jsx

```diff
@@ -43,7 +43,12 @@ export class BrowseControlBar extends React.Component {
         return (
             <React.Fragment>
                 {showActions && !this.isRootNode() &&
-                <DisplayActions target="tableHeaderActions" context={{path: path}} render={ButtonRenderer}/>}
+                <React.Fragment>
+                    <DisplayAction actionKey="createContent" context={{path: path}} render={ButtonRenderer}/>
```

Comment:
why not adding createContent in headerPrimaryActions ?

Reply:
I'll do a PR in content-editor and do it there

Reply:
ok